packages/codebytes/src/codeByteEditor.tsx

```diff
@@ -70,7 +71,7 @@ export const CodeByteEditor: React.FC<CodeByteEditorProps> = ({
   }, []);
 
   return (
-    <EditorContainer bg="black" as="main" isIFrame={isIFrame}>
+    <EditorContainer bg="black" isIFrame={isIFrame} {...rest}>
```

Comment:
i actually think we should probably remove the isIframe state and have the nextjs handle those, but i didn't want to break your pr @BandanaKM

Reply:
it also lets us remove a prop :)

Reply:
i went ahead and pushed it but if we want i can revert the commit